### PR TITLE
Testability statements section 5.5.4

### PR DESCRIPTION
**Pull request by KCP1023**
Testability Statements for section 5.5.4

### CODE REVIEW

**Review by Therecanbeonlyone1969 — COMMENTED** [review]
@KCP1023 you need to update your branch from main. You are 6 commits behind the main branch.

File: .idea/workspace.xml
Comment (Therecanbeonlyone1969) [inline .idea/workspace.xml:1]: @KCP1023 remove this file and folder from your branch, please

File: core/baseline-core-v1.0-psd01.md [hunk 1]
```diff
@@ -4704,6 +4722,34 @@ In the context of BPI interoperability, the BPI export operation MUST provide at
...
+
+Test Steps:
+
+1. Initiate the BPI export operation from the exporting BPI system and review the exported data.
+
+Expected Result:
+1. The State Object is included in the exported data. 
+2. The Zero-Knowledge Proof(s) of Correctness are part of the exported data. 
+3. All public input data for validating the Zero-Knowledge Proofs is provided. 
+4. Private input data for the Zero-Knowledge Proofs is included securely.
```
Comment (Therecanbeonlyone1969) [inline core/baseline-core-v1.0-psd01.md:4742]: @KCP1023 to verify a proof private data is not required, just the public inputs, th zkp, the verification key

Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:4742]: Also, can you remove line 4714 in the requirement? That was included in error.

File: core/baseline-core-v1.0-psd01.md [hunk 2]
```diff
@@ -4704,6 +4722,34 @@ In the context of BPI interoperability, the BPI export operation MUST provide at
...
+1. The State Object is included in the exported data. 
+2. The Zero-Knowledge Proof(s) of Correctness are part of the exported data. 
+3. All public input data for validating the Zero-Knowledge Proofs is provided. 
+4. Private input data for the Zero-Knowledge Proofs is included securely. 
+5. The Verification Keys for the Zero-Knowledge Proofs are accessible. 
+6. The prover system specifications for Zero-Knowledge Proofs are provided. 
+7. The validation URI for third-party verification is included. 
+8. The lock commitment of the current state object is part of the export. 
+9. Public input data for the lock commitment is included. 
+10. Private input data for the lock commitment is included securely.
```
Comment (Therecanbeonlyone1969) [inline core/baseline-core-v1.0-psd01.md:4748]: remove this line as well

File: core/baseline-core-v1.0-psd01.md (hunk 1 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:4742]: And line 4720

File: core/baseline-core-v1.0-psd01.md [hunk 3]
```diff
@@ -4746,13 +4816,62 @@ Bi- or Multi-directional BPI services in the context of BPI interoperability MUS
...
+
+* Two or more BPI systems are set up for bi- or multi-directional BPI services.
+* All BPI systems involved are configured for interoperability.
+* The BPI Subjects and Participants are properly registered in their respective systems.
+
+Test Steps:
+
+1. Create a State Synchronization and Advancement Predicate from one BPI system.
+2. Update the State Synchronization and Advancement Predicate from another BPI system.
+3. Launch an IVSM (Interoperable Verification Smart Master) from one of the BPI systems.
```
Comment (Therecanbeonlyone1969) [inline core/baseline-core-v1.0-psd01.md:4831]: IVSM = Interoperability Virtual State Machine ... no need to spell it out since it is defined already above

Comment (chaals) [reply core/baseline-core-v1.0-psd01.md:4831]: It would be nice if odd terms and acronyms could link to where they are defined.

**Review by Therecanbeonlyone1969 — CHANGES_REQUESTED** [review]
@KCP1023 please, make the requested changes.  Then good to approve.

**Review by Ybittan — APPROVED** [review]
lgtm

**Pull request merged**